tests/Api/PlaylistsTest.php

```diff
@@ -365,9 +362,25 @@ public function testGetScreensList(): void
         $this->assertResponseIsSuccessful();
         $this->assertResponseHeaderSame('content-type', 'application/ld+json; charset=utf-8');
         $this->assertJsonContains([
-            '@context' => '/contexts/Screen',
-            '@id' => '/v1/screens',
+            '@context' => '/contexts/ScreenCampaign',
+            '@id' => '/v1/screen-campaigns',
+            '@type' => 'hydra:Collection',
+        ]);
+    }
+
+    public function testSharedPlaylists(): void
```

Comment:
I think what we are missing in terms of tests, is that we can CRUD through the API. And that things are as expected after the different operations.